hack/generate-yamls.sh

```diff
@@ -54,6 +54,7 @@ readonly SERVING_CRD_ALPHA_YAML=${YAML_OUTPUT_DIR}/serving-alpha-crds.yaml
 readonly SERVING_ALPHA_YAML=${YAML_OUTPUT_DIR}/serving-pre-1.14.yaml
 readonly SERVING_CRD_BETA_YAML=${YAML_OUTPUT_DIR}/serving-beta-crds.yaml
 readonly SERVING_BETA_YAML=${YAML_OUTPUT_DIR}/serving-post-1.14.yaml
+readonly SERVING_CORE_BETA_YAML=${YAML_OUTPUT_DIR}/serving-core-post-1.14.yaml
```

Comment:
This file is outdated, please merge latest master and add this file to the list at the bottom of this file

Reply:
this actually depends on whether this file should be included as part of serving release, @mattmoor  please take a look

Reply:
i've updated from master, but waiting for the reply on this thread on whether to update the serving release yamls

Reply:
We shouldn't add more files to the release, if we can avoid it.